public/index.html

```diff
@@ -25,7 +25,7 @@ <h1 class="alignright">MapGT</h1>
         </div>
     
         <div class="section toggleButton">
-            <button class="btn btn--floor1" onclick="tog1()">1st</button>
+            <button class="btn btn--floor1 active" id="removeactive" onclick="tog1()">1st</button>
```

Comment:
I don't like that one of the buttons has an id while the other doesn't – it's kinda weird. Why don't we just get elements by class name `btn`, remove the `active` class from all of them, and add the class to the button that was just clicked? I think it'd be a lot more elegant and extensible in the future if we add more buttons